src/coreclr/jit/emitarm64.cpp

```diff
@@ -10942,6 +11025,9 @@ void emitter::emitIns_R_R_R(instruction     ins,
             fmt = IF_SVE_HP_3A;
             break;
 
+        case INS_sve_prfb:
```

Comment:
Was this intended?

Reply:
Nope, that should be in the next PR. Thanks for catching that.